components/tooltip/tooltip-help.js

```diff
@@ -103,12 +107,14 @@ class TooltipHelp extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			'd2l-body-small': !this.inheritFontStyle
 		};
 		return html`
-			<button id="d2l-tooltip-help-text" class="${classMap(classes)}" type="button">
+			<button id="d2l-tooltip-help-text" class="${classMap(classes)} d2l-skeletize" type="button">
```

Comment:
Suggest adding this to the `classes` so there isn't a leading space when `d2l-body-small` isn't there:

```javascript
const classes = {
	'd2l-body-small': !this.inheritFontStyle,
	'd2k-skeletize': true
};
```